prowler/compliance/azure/mitre_attack_azure.json

```diff
@@ -0,0 +1,2496 @@
+        },
+        {
+          "AzureService": "Azure Policy",
+          "Category": "Protect",
+          "Value": "Partial",
+          "Comment": "This control may provide recommendations to enable Azure services that limit access to cloud services. Several Azure services and controls provide mitigations against cloud service discovery."
+        },
+        {
+          "AzureService": "Cloud App Security Policies",
+          "Category": "Detect",
+          "Value": "Partial",
+          "Comment": "This control can detect anomalous user activity that may be associated with cloud service discovery. Relevant alert is \"Unusual file share activty (by user)\"."
+        }
+      ]
+    },
+    {
+      "Name": "Cloud Storage Object Discovery",
+      "Id": "T1619",
+      "Tactics": [
+        "Discovery"
+      ],
+      "SubTechniques": [],
+      "Platforms": [
+        "IaaS"
+      ],
+      "Description": "Adversaries may enumerate objects in cloud storage infrastructure. Adversaries may use this information during automated discovery to shape follow-on behaviors, including requesting all or specific objects from cloud storage. Similar to File and Directory Discovery on a local host, after identifying available storage services (i.e. Cloud Infrastructure Discovery) adversaries may access the contents/objects stored in cloud infrastructure.",
+      "TechniqueURL": "https://attack.mitre.org/techniques/T1619/",
+      "Checks": [],
+      "Attributes": []
+    },
+    {
+      "Name": "Network Service Discovery",
+      "Id": "T1046",
+      "Tactics": [
+        "Discovery"
+      ],
+      "SubTechniques": [],
+      "Platforms": [
+        "Containers",
+        "IaaS",
+        "Linux",
+        "Network",
+        "Windows",
+        "macOS"
+      ],
+      "Description": "Adversaries may attempt to get a listing of services running on remote hosts and local network infrastructure devices, including those that may be vulnerable to remote software exploitation. Common methods to acquire this information include port and/or vulnerability scans using tools that are brought onto a system.",
+      "TechniqueURL": "https://attack.mitre.org/techniques/T1046/",
+      "Checks": [
+        "network_http_internet_access_restricted",
+        "network_rdp_internet_access_restricted",
+        "network_ssh_internet_access_restricted",
+        "network_udp_internet_access_restricted",
+        "network_watcher_enabled",
+        "networkfirewall_in_all_vpc"
```

Comment:
```suggestion
```
"networkfirewall_in_all_vpc" is not a check for azure

Reply:
Deleted